importing-exporting-data.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<meta charset="UTF-8">
+<link rel="stylesheet" type="text/css" href="styles.css"> <!--css sheet-->
+<script src="scripts/jquery.js"></script> <!--jquery library-->
+<script src="scripts/seasonConfigScript.js"></script> <!--js script-->
+<script src="scripts/eventDataScript.js"></script> <!--js script-->
+<script src="scripts/teamDataScript.js"></script> <!--js script-->
+<script src="scripts/commonMethodHelpers.js"></script> <!--js script-->
+<script src="scripts/importing-Exporting-Config.js"></script> <!--js script-->
+<html>
+    <head>
+        <link rel="shortcut icon" href="images/club_logo.png" type="image/x-icon" />
+        <title>FRC 6321 Scouting</title>
+        <!--Importing fonts-->
+        <link href="https://fonts.googleapis.com/css?family=Ubuntu&display=swap" rel="stylesheet">
+    </head>
+    <body id="background">
+        <button style = "float:left" onclick = "openNav()" class="r3-button">Navigation Menu</button>
+        <div>
+            <h1 id = "title">FRC 6321 Scouting</h1>
+        </div>
+        <div id = "mySidenav" class="sidenav">
+            <a href="javascript:void(0)" class="closebtn"
+            onclick = "closeNav()">&times;</a>
+            <a href="index.html">Home</a>
+            <a href="season-config.html">Season Configuration</a>
+            <a href="event-data.html">Event Data</a>
+            <a href="team-data.html">Team Data</a>
+            <a href="importing-exporting-data.html">Importing and Exporting Data</a>
+        </div>
```

Comment:
I have a suspicion this `nav` isn't updated everywhere.

After this PR, while waiting to get the demo with the team done, look into using Jekyll to include this top level stuff. (I can link you to the docs and some examples if desired).

Reply:
Yeah, I've noticed that too: when I go to the Team Data page the text of the nav bar button reverts to times new roman. I remember you talking me through how to do this on the slack a few months ago but further docs and links would be appreciated.

Reply:
Further docs:
https://jekyllrb.com/docs/

and links:
https://jekyllrb.com/
https://github.com/mshafer1/themepuzzler

Jekyll is the tool that gh-pages uses to build your site anyways.
If you setup with an `_includes` folder and put `nav.html` in there,
then each of your pages can have
`{% include nav.html %}`
to have the contents of `nav.html` put in at that point (kind of like a sub vi, but in-lined when the site is "built" into static pages).

Jekyll is a very powerful tool built for static websites and blogs - but even just having a basic layout (see the docs) or a few files you can include (like the nav), goes a long ways towards making things that should be updated just once but affect everywhere.